lib/edit-foods.js

```diff
@@ -0,0 +1,14 @@
+$('document').ready( function() {
+  $('body').on('click', '.food-row td', function() { 
+    var $el = $(this);
```

Comment:
@wlffann Could you use a different variable name "el" is a little confusing to me."